### PR TITLE
[Refactor] LunarVim Info UI

### PR DESCRIPTION
**Pull request by PasiBergman**
# Description

Make LunarVim Info UI more compact.

- Run command `:LvimInfo` on LS activated filetype buffer

### CODE REVIEW

File: lua/core/info.lua [hunk 1]
```diff
@@ -4,15 +4,20 @@ local null_ls_handler = require "lsp.null-ls"
 local indent = "  "
 
 M.banner = {
+  indent .. "Press q to close",
```
Comment (kylo252) [inline lua/core/info.lua:7]: My only real complaint is this. I think it looks off. I tried giving it a big offset but I'm not sure it's that much better anyway
```lua
  local function get_adaptive_dismiss_hint(window_width)
    local msg = " "
    local hint = "Press q to dismiss"
    local padding = 4
    for i = 1, (window_width - (string.len(hint) + padding)) do
      msg = msg .. " "
    end
    return { "", msg .. hint, "" }
  end

  vim.list_extend(buf_lines, get_adaptive_dismiss_hint(math.floor(vim.o.columns * 0.7)))
```

Maybe we should see how Telescope embeds it in the frame.

File: lua/core/info.lua [hunk 2]
```diff
@@ -126,42 +133,47 @@ function M.toggle_popup(ft)
...
     "",
   }
   vim.list_extend(buf_lines, header)
 
   local lsp_info = {
-    "Associated language-server: " .. client.name,
-    indent .. "* Active: " .. tostring(is_client_active) .. ", id: " .. tostring(client.id),
-    indent .. "* Formatting support: " .. tostring(client.resolved_capabilities.document_formatting),
-    indent .. "* Capabilities list: " .. table.concat(vim.list_slice(client_enabled_caps, 1, num_caps / 2), ", "),
+    indent .. "Language Server",
```
[the remark this answers is not in the document]
Comment (PasiBergman) [reply lua/core/info.lua]: Renamed line to suggested one - line is not too long.

File: lua/core/info.lua (hunk 1 above)
Comment (PasiBergman) [reply lua/core/info.lua]: Removed "Press q to close"

**Comment by ChristianChiarulli** [comment]
@kylo252 are we happy with this yet?

**Review by kylo252 — APPROVED** [review]

**Pull request merged**